drivers/overlay/overlay.go

```diff
@@ -431,6 +441,20 @@ func (d *Driver) Get(id string, mountLabel string) (s string, err error) {
 		return "", err
 	}
 
+	if _, err = os.Stat(string(lowers)); err != nil {
```

Comment:
I don't see `lowers` being modified after it's read from the file.  Running 'go test' under strace is still showing me calls to `stat()` with inputs like `l/BQQTO2FOMZN5GFRI6JBQFXF2DP:l/TNXWF7RJCERTMLWZ75EI73J54H:l/64MPZFDTMQ623P5YJIZGZFSVU4:l/SEFLQHUOKLMFAOTKZJODYHGTTF`.